src/AppBundle/Membership/MembershipRequest.php

```diff
@@ -120,14 +120,24 @@ public function __construct()
         $this->conditions = false;
     }
 
-    public static function createWithCaptcha(string $recaptchaAnswer = null)
+    public static function createWithCaptcha(string $recaptchaAnswer = null): self
     {
         $dto = new self();
         $dto->recaptcha = $recaptchaAnswer;
 
         return $dto;
     }
 
+    public function setEmailAddress(string $emailAddress)
+    {
+        $this->emailAddress = mb_strtolower($emailAddress);
```

Comment:
So I think we need to store the canonical email address in a separate field when checking for the uniqueness of the user account.